charts/greptimedb-cluster/templates/grafana-dashboards-sidecar.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if or .Values.grafana.enabled .Values.dashboards.enabled }}
```

Comment:
Maybe the original name `grafana-dashboards-configmap.yaml` will be better? I always thought the sidecar was for pods.